cpp/ps0005/ps0005_solv.cpp

```diff
@@ -4,82 +4,75 @@ using namespace std;
 
 int main()
 {
-    int x,y,result;
-    char z;
+    // prompting user for 2 operands and an operator 1.5/2
+    float num1, num2, result;
+    char op;
 
+    cout << "Enter the two operands : \n";
+    cin >> num1 >> num2;
 
-    cout<<" Enter the two operands : \n ";
-    cin>>x>>y ;
+    cout << "Enter the two operator + - * / % : \n";
+    cin >> op;

```

Comment:
- [x] declared variables to store the user input
- [ ] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input